### DISCUSSION TITLE
Add caching to Preview Cleanup

### DISCUSSION THREAD

**Discussion by WolfInABowl** [opentoonz/opentoonz#6401] [Ideas]
Here's a scenario:
You are using OT's Cleanup tools to trace several drawings.
You turn on `Preview Cleanup` and OT generates the preview for your current settings for drawing 1.
You switch to drawing 2 and OT does the same for that drawing.
You switch back to drawing 1, OT will regenerate the preview for that drawing even though no settings have been changed.

This makes `Preview Cleanup` slower than it could otherwise be.

**Comment by RodneyBaker** [comment]
Converting this to Discussion so others can explore information and consider the best way forward.
As this request is five years old it is possibly partially or fully implemented.
